### ISSUE TITLE
Performance on PascalVOC with default config in voc8.res50v3+.CPS

### ISSUE THREAD

**Issue by Haiduongcable** [charlesCXK/TorchSemiSeg#51]
Hello, I am using 1 Telsa P100 16gb with batchsize 4, with your default config. 
I only reach about 66% mIOU, different with your report in paper and your log: 73.28

**Comment by charlesCXK** [comment]
Hi, if you only use 1 GPU, the learning rate should be adjusted according to the batch size (e.g. use the linear scaling rule for lr - bs).
However, we recommend using 4 GPUs since the batch size is very important in the semantic segmentation task.

**Comment by Haiduongcable** [comment]
Thank you.
I change learning rate decrease to about 0.001 (1/5 your default learning rate) with batchsize 4.
But I only got 71% mIOU. 
I decrease learning rate to 0.0005 (1/10 your default learning rate) with batchsize 4, but result is only stable between 70.5%-71% mIOU.

**Comment by lijiwen97** [comment]
> Hello, I am using 1 Telsa P100 16gb with batchsize 4, with your default config. I only reach about 66% mIOU, different with your report in paper and your log: 73.28

Hello, the log file after my training is red, which cannot be called in the verification process. Have you ever encountered such a situation? (I also train with a single GPU )

**Comment by charlesCXK** [comment]
@lijiwen97 
Hi, I think this issue has nothing to do with the number of GPUs. It seems that something was wrong when saving the checkpoint, e.g., the disk was full at that time.

**Comment by qwhdhdue** [comment]
Hello, I am using 4gpu, and your configuration lr=0.0025, batchsize is 8 for training.  But only reached 67% mIOU.
